packages/entities/entities-shared/src/components/entity-base-config-card/EntityBaseConfigCard.vue

```diff
@@ -176,6 +243,30 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  /**
+   * The label text to show for the select Format element
+   */
+  label: {
+    type: String,
+    default: 'Format: ',
+    required: false,
+  },
+  /**
+   * Boolean to determine whether to display the documentation button
+   */
+  hideConfigCardDoc: {
+    type: Boolean,
+    default: false,
+    required: false,
+  },
```

Comment:
Do we need another prop to determine whether to display the documentation button? We already have `configCardDoc`, so I think we can control the visibility of the documentation button by checking if `configCardDoc` is undefined or an empty string.

Reply:
Sorry I wasn't clear before. What I meant was since we already have `configCardDoc`, do we still need `hideConfigCardDoc`? We can control the button's visibility by checking `configCardDoc` and remove `hideConfigCardDoc`.

Reply:
Added both just in case to not run into issue if someone didnt provide the doc url and still wanted to show the button. I can remove `hideConfigCardDoc` if it's not making sense here.

Reply:
I will remove the `hideConfigCardDoc`.